.ci/jenkins/Jenkinsfile.post-release

```diff
@@ -116,6 +116,31 @@ pipeline {
                 }
             }
         }
+       stage('update kie-benchmarks main to next SNAPSHOT version') {
+            steps {
+                script {
+                    String prLink = null
+                    String kieBenchmarksRepo = 'kie-benchmarks'
+                    String commitMsg = 'bumped up kie-version to next SNAPSHOT'
+                    String prBranch = 'kie-benchmarks-update'
+                    String targetBranch = 'main'
+                    dir("$kieBenchmarksRepo-bot"){
```

Comment:
I think there should be kieBenchmarksRepo - instead of kieBenchmarksRepo-bot the so jenkins will create a new dir with kieBenchmarksRepo name and execute all steps inside